src/settings/Settings.tsx

```diff
@@ -173,6 +173,11 @@ export interface IBaseSetting<T extends SettingValueType = SettingValueType> {
 
     // Whether the setting should have a warning sign in the microcopy
     shouldWarn?: boolean;
+
+    /**
+     * Whether the setting should be exported in a rageshake report.
+     */
```

Comment:
@Half-Shot please could you document what it means if this is unset? Presumably it's the same as setting it to `false`?